cmake/add_project_darknet.cmake

```diff
@@ -17,6 +17,9 @@ endif()
 
 if( VIAME_ENABLE_CUDA )
   FormatPassdowns( "CUDA" VIAME_CUDA_FLAGS )
+  set( VIAME_DARKNET_CUDA_FLAGS 
+    -DCUDA_TOOLKIT_ROOT_DIR:PATH=${CUDA_TOOLKIT_ROOT_DIR}
```

Comment:
This CUDA path should already be included in the above macro, were you seeing issues with it being passed down?

Reply:
I observed that it wasn't getting passed down at all. Fletch had the same issue with building darknet, and this is the same exact patch it needed.

Reply:
It should be in the above macro, which searches the CMAKE cache for anything starting with CUDA_* and then passes them down, but can leave in for now and revist later